### PR TITLE
Requirements for Python and cmake and their steps to install in case someone has some exception in their device :)

### PR DESCRIPTION
**Pull request by yash982000**

### CODE REVIEW

**Comment by jounathaen** [comment]
You are correct, python is a requirement and should be listed. Out of curiosity: On which system are you working, which doesn't have python installed right away?

I do not like the paragraph you added. It is too verbose. I think mentioning python in the requirements should be sufficient. I guess, if you are at a point where you use unikernels, you don't need to be teached how to install some packages on ubuntu.

**Comment by yash982000** [comment]
@jounathaen I am working on Ubuntu Linux 20.04. Still, I discovered that sometimes it gives an error about python not being installed or python3 needed. My other two team members faced this same thing.

True. I am still not comfortable with unikernels. I added this for providing additional help to some of the beginners who might have taken this as a college project and face this. Rest , whatever you say is true. Let me know if you want me to change something or add something new by rephrasing. :+1:

File: README.md [hunk 1]
```diff
@@ -124,6 +125,7 @@ cmake-3.7.2-Linux-x86_64.tar.gz         100%[===================>]  29,26M  3,74
 -- Local CMake v3.7.2 installed to cmake/cmake-3.7.2-Linux-x86_64
 -- Next time you source this script, no download will be necessary
 ```
+##### NOTE:-If you do not have Python installed, it will give an error. To avoid this, run this command *"sudo apt-get install python"* or *"sudo apt-get install python3"* as per your requirement. ALso, after this , even cmake can give some error. You can avoid it by running this command "sudo snap install cmake". Hope this will run for all perfectly :)
```
Comment (jounathaen) [inline README.md:128]: As described previously, I think this paragraph should be reverted.

**Review by jounathaen — CHANGES_REQUESTED** [review]

**Comment by jounathaen** [comment]
Would you maybe describe the python problems in a new issue? I have the feeling that the real problem is something else. I'm pretty sure that on a regular ubuntu installation python is already preinstalled

File: README.md (hunk 1 above)
Comment (yash982000) [reply README.md:128]: @jounathaen Sure, no issues. Should I remove it completely?

**Comment by yash982000** [comment]
> Would you maybe describe the python problems in a new issue? I have the feeling that the real problem is something else. I'm pretty sure that on a regular ubuntu installation python is already preinstalled

I will open a new issue from starting regarding the python installation and requirement. For now, is it fine to merge the readme changes of Python in requirement and close this issue and PR?

**Comment by meet-akbari** [comment]
Hello @jounathaen I think whatever @yash982000 is saying correct at his side. He want to convey to you if someone has newly installed ubuntu and he/she is trying to install libhermit in their ubuntu system then he/she will may get the error as @yash982000 has mentioned because the same issue I have faced. So, in future anyone like us not face any issue regarding it.

File: README.md (hunk 1 above)
Comment (jounathaen) [reply README.md:128]: Yes, I don't think the Information belongs here.

**Comment by jounathaen** [comment]
Ok, the problem on Ubuntu was, that python did not default to python3. I changed the script in https://github.com/hermitcore/libhermit/commit/7977ae391040dc2209a65f43817cc2d9d27dae71 
It should now work on Ubuntu 20.04 out-of-the-box. (except for nasm and cmake which need to be installed)

File: README.md [hunk 2]
```diff
@@ -44,6 +44,7 @@ the HermitCore kernel and applications you need:
  * Netwide Assember (NASM)
  * recent host compiler such as GCC
  * HermitCore cross-toolchain, i.e. Binutils, GCC, newlib, pthreads
+ * Python
```
Comment (jounathaen) [inline README.md:47]: Actually it must be "Python3"

**Review by jounathaen — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (yash982000) [reply README.md:128]: @jounathaen Okay!!

File: README.md (hunk 2 above)
Comment (yash982000) [reply README.md:47]: @jounathaen Exactly!! I changed it!!

**Comment by yash982000** [comment]
@jounathaen I did all the required changes, you can now merge this. I will check the Python3 problem again if it has resolved or not and will open it in a new issue.

**Comment by jounathaen** [comment]
Alright, thanks for reporting and your efforts! :+1:

**Comment by yash982000** [comment]
> Alright, thanks for reporting and your efforts! 👍

That's my pleasure :) Will be working on this project for around 1 more month and will point out some issues/features if any. Looking forward to making this project more comprehensive and helpful for everyone!!

**Pull request merged**